src/Tools/Microsoft.dotnet-openapi/src/Microsoft.dotnet-openapi.csproj

```diff
@@ -31,5 +31,10 @@
       <_Parameter2>$(NewtonsoftJsonVersion)</_Parameter2>
       <_Parameter3>NSwagCSharp;NSwagTypeScript</_Parameter3>
     </AssemblyAttribute>
+    <AssemblyAttribute Include="Microsoft.DotNet.Openapi.Tools.Internal.OpenApiDependencyAttribute">
+      <_Parameter1>Microsoft.OpenApi.Kiota.ApiDescription.Client</_Parameter1>
```

Comment:
Referencing only this package, will updated projects using `KiotaCSharp` or `KiotaTypeScript` be able to successfully use a generated client❔

Reply:
what would be the recommended approach from your point of view? add those dependencies to the APIDescription package?
Or add those dependencies to the target through the same mechanism we're adding the API description package?
I prefer the later as it provides more control to users over which dependencies and versions dependencies they bring to their project should they want to change things.

Also, how are the JS/TS/npm packages added? are they being wrapped in a nuget package? or is there a possibility to add a reference to an npm package? if so how?

Reply:
added the dependencies for dotnet. waiting on @bradygaster 's input for the typescript ones